logs_analyzer/chapter2/scala/src/main/scala/com/databricks/apps/logs/ApacheAccessLog.scala

```diff
@@ -0,0 +1,35 @@
+package com.databricks.apps.logs
+
+/**
+  * An entry of Apache access log.
+  */
+case class ApacheAccessLog(ipAddress: String,
```

Comment:
Seems I have seen similar code several times before. Is there any way to DRY this up?